app/services/github_service.rb

```diff
@@ -11,12 +11,13 @@ class << self
 
     ##
     # This method get the authorization url which authorizes the post editor app
-    # to use a user's GitHub account. The scope is public_repo so that we're able
+    # to use a user's GitHub account. The scopes public_repo is used so that we're able
     # to make changes to the msoe-sse/mseo-sse.github.io repository which requires
-    # access a user's public repositories
+    # access a user's public repositories. Also, the scope read:org is used so that
```

Comment:
Did you mean "access _to_ a user's public repositories"? Just checking...

Reply:
Thank You :)